Update execution containers with info on support for Windows images (#1327)
OctopusDeploy/docs: docs/projects/steps/execution-containers-for-workers/index.md

It can be beneficial to build your own custom Docker image when using execution containers, particularly when you wish the image size to be as small as possible.

#### Supported Windows base images

> For Windows images, we recommend using no older than the `ltsc2019` image as a base image. Octopus does not support images which are older than `ltsc2019`, and while steps can still be run on containers based on these images, you may run into unexpected issues.

Reviewer: ```suggestion
For Windows images, we recommend using a base image no older than the `ltsc2019` image. Octopus does not support images that are older than `ltsc2019`, and while containers based on these images can still run steps, you may run into unexpected issues.
```
